src/res/translation/wininstaller/se.nsi

```diff
@@ -0,0 +1,37 @@
+; Swedish translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_SWEDISH} \
+    "Skapa skrivbordsgenväg"
+
+LangString INVALID_FOLDER_MSG ${LANG_SWEDISH} \
+    "Mappen finns redan. Ange en ny målmapp."
+
+LangString RUNNING_APP_MSG ${LANG_SWEDISH} \
+    "${APP_NAME} körs. Stäng den och kör installationen igen."
+
+LangString OLD_WRONG_VER_FOUND ${LANG_SWEDISH} \
+    "Vi upptäckte en gammal version av ${APP_NAME} i din 32-bitars programmapp. Vi rekommenderar starkt att du tar bort den innan du installerar en ny version av ${APP_NAME}. Vill du ta bort den nu?"
+
+LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_SWEDISH} \
+    "Om du fortsätter utan att ta bort den finns det risk att installationen inte kommer att fungera. Är du säker på att du inte vill ta bort den gamla versionen?"
+
+LangString OLD_VER_REMOVE_FAILED ${LANG_SWEDISH} \
+     "Vi kunde tyvärr inte avinstallera den gamla versionen. Vi försöker installera den nya, men du kan också trycka på Avbryt och försöka ta bort den gamla versionen själv."
+
+LangString ASIO_DRIVER_HEADER ${LANG_SWEDISH} \
+     "ASIO-ljuddrivrutin"
+
+LangString ASIO_DRIVER_SUB ${LANG_SWEDISH} \
+     "För att kunna använda Jamulus behöver du en ASIO-ljuddrivrutin"
+
+LangString ASIO_DRIVER_EXPLAIN ${LANG_SWEDISH} \
+     "Jamulus behöver en ASIO-ljuddrivrutin för att ge ljud med låg latens. Mer information:"
+
+LangString ASIO_DRIVER_MORE_INFO ${LANG_SWEDISH} \
+     "Mer information om ASIO hittar du på jamulus.io"
+
+LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_SWEDISH} \
+     "https://jamulus.io/wiki/Installation-for-Windows#asio"
+
+LangString ASIO_EXIT_NO_DRIVER ${LANG_SWEDISH} \
+     "${APP_NAME} behöver en ASIO-ljuddrivrutin för att fungera, men vi kunde inte hitta någon på din dator. Du bör installera en som ASIO4ALL (Mer information på jamulus.io under "Installation for Windows"). Vill du fortfarande fortsätta med installationen av $ {APP_NAME} först?"
```

Comment:
This seems to break the build as the translated string contains double quotes. I don't know the nsi language, but it might either need escaping (`\"?) or one might want to try single quotes which should be less invasive.

Reply:
Proper [quoting syntax seems](https://nsis.sourceforge.io/How_can_I_use_quotes_in_a_string) to be `$\"`.

```suggestion
     "${APP_NAME} behöver en ASIO-ljuddrivrutin för att fungera, men vi kunde inte hitta någon på din dator. Du bör installera en som ASIO4ALL (Mer information på jamulus.io under $\"Installation for Windows$\"). Vill du fortfarande fortsätta med installationen av $ {APP_NAME} först?"
```